indigo_app/views/places.py

```diff
@@ -100,12 +100,133 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class PlaceDetailView(PlaceViewBase, AbstractAuthedIndigoView, ListView):
+class PlaceDetailView(PlaceViewBase, AbstractAuthedIndigoView, TemplateView):
     template_name = 'place/detail.html'
+    tab = 'overview'
+    js_view = 'PlaceWorksView'
```

Comment:
Should be `PlaceDetailView` (which is the same as not specifying `js_view` at all.